pkg/utils/utils.go

```diff
@@ -148,8 +149,8 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState) error {
 		for _, iface := range newState.Spec.Interfaces {
 			if iface.PciAddress == ifaceStatus.PciAddress {
 				configured = true
-				if iface.EswitchMode == sriovnetworkv1.ESWITCHMODE_SWITCHDEV && ClusterType == ClusterTypeOpenshift {
-					// Skip sync for openshift, MCO will inject a systemd service to config switchdev devices.
+				if SkipConfigVf(iface, ifaceStatus) {
```

Comment:
This removes `ClusterType == ClusterTypeOpenshift` conditional check.
@e0ne  Do you see this will be a problem for k8s switchdev configuration?